Assets/Scripts/TapeInformation.cs

```diff
@@ -0,0 +1,8 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TapeInformation : MonoBehaviour
```

Comment:
Kind of confused why we need both a `TapeInformation` and `TapeSO` classes? What's the difference between them?

Reply:
I dont know if I can add a Scriptable Object straight up as a component to a GameObject(I tried but it didn't work). So I created a script TapeInformation with the TapeSO class as a var, and attached the script TapeInformation as a component to the Tape GameObject

Reply:
Okay, read up on scriptable objects, this makes more sense now :)